test/feedback/components/FeedbackSubmitted.unit.spec.jsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import enzyme from 'enzyme';
+import { expect } from 'chai';
+
+import FeedbackSubmitted from '../../../src/js/feedback/components/FeedbackSubmitted';
+
+describe('<FeedbackSubmitted/>', () => {
+
+  it('should render with follow-up message', () => {
+    const wrapper = enzyme.shallow(<FeedbackSubmitted shouldSendResponse/>);
+    const text = wrapper.text();
+    expect(text.includes('We\'ll get back to you soon.')).to.be.true;
```

Comment:
Oh, I think the `'` in `We'll` should be a curly apostrophe (`’`).

Reply:
Good call, will fix